apps/admin/src/utils/http.ts

```diff
@@ -0,0 +1,33 @@
+import ky from 'ky';
+import { getAccessToken } from './api';
+
+export const http = ky.create({
+  headers: {
+    'Content-Type': 'application/json',
+  },
+  credentials: 'include',
+  timeout: 10000,
+  hooks: {
+    beforeRequest: [
+      (request) => {
+        const accessToken = getAccessToken();
+        if (accessToken) {
+          request.headers.set('Authorization', `Bearer ${accessToken}`);
+        }
+      },
+    ],
+  },
+});
+
+export const request = async <T>(
```

Comment:
```suggestion
/**
 * @deprecated http 유틸리티를 사용하세요.
 */
export const request = async <T>(
```

request 래핑은 기존 기존 레거시 호환하기 위함인가요??

Reply:
레거시 호환은 아니고 제가 테스트 하는 과정에서 생긴 불필요 코드입니다.
http 유틸리티만 사용하도록 변경하겠습니다.